benchmark/framework/aea_test_wrapper.py

```diff
@@ -37,19 +37,27 @@
 class AEATestWrapper:
     """A testing wrapper to run and control an agent."""
 
-    def __init__(self, name: str = "my_aea", skills: List[dict] = None):
-        """Make an agency with optional name and skills."""
+    def __init__(self, name: str = "my_aea", skills: List[Dict[str, Skill]] = None):
+        """
+        Make an agency with optional name and skills.
+
+        :param name: name of the agent
+        :param skills: dict of skills to add to agent
+        """
         self.skills = skills or []
         self.name = name
 
         self.aea = self.make_aea(
-            self.name, [self.make_skill(**skill) for skill in self.skills]
+            self.name, [self.make_skill(**skill) for skill in self.skills]  # type: ignore
```

Comment:
why type ignore?

Reply:
I use type ignore in some complex cases, for example mypy can not get in dict structure of this config, and show error.